packages/import-bundle/src/compartment-wrapper.js

```diff
@@ -51,6 +52,15 @@ export function wrapInescapableCompartment(
     // there are details to work out.
     c.globalThis.Compartment = NewCompartment;
 
+    for (const prop of Object.keys(inescapableGlobalProperties)) {
+      Object.defineProperty(c.globalThis, prop, {
+        value: inescapableGlobalProperties[prop],
+        writable: true,
+        enumerable: false,
+        configurable: true,
+      });
+    }
```

Comment:
This could be more succinctly `Object.defineProperties(object, Object.getOwnPropertyDescriptors(inescapableGlobalProperties))`.

Reply:
Is that going to be safe (or, if safe, what we want) if somebody hardens the `inescapableGlobalProperties` object prior to getting here?

Reply:
I think it’s a feature to allow `inescapableGlobalProperties` to communicate the full, intended property descriptors.

Reply:
*However*, SES Compartment endowments are applied with `Object.assign`. We have previously discussed adding a `globalProperties` option that would apply descriptors.

Reply:
I'd be wary of having the property descriptors on the properties of the `inescapableGlobalProperties` argument magically act as hidden arguments of their own, especially if somebody in the call chain does a harden without realizing where the object is going.